src/experiments/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "downlevelIteration": true,
+    "target": "es5",
+    "strictNullChecks": true,
+    "noImplicitAny": true,
+    "lib": ["dom", "esnext", "es6"],
+
+    "suppressImplicitAnyIndexErrors": true,
```

Comment:
Can we solve the issues instead of suppressing?

```suggestion
```

Reply:
These errors occur any time we do `element[FOO_BAR_PROP]` or `win[BAR_BAZ_PROP_NAME]` etc. We do those all the time and afaict this wouldn't catch anything meaningful. We _could_ solve it with a bunch of ugly casting, but I don't think that's a win tbh. WDYT?

Reply:
When do we do `win[BAR_BAZ_PROP_NAME]`? I was seeing us adding non-standard objects to the window and/or dom nodes for caching etc.

In those cases, the fix is to augment the global.

Another time I saw it was for non-standard accesses for legacy browsers. Those likely deserve typecasting and comments, and aren't that common.

The last case I noticed was when we use a string for lookup, i.e. `styles['justifyContent']`. I believe we had done this to ensure Closure doesn't mangle the name. For these cases we can simply rename to `styles.justifyContent`.

Reply:
GitHub search is terrible, but in your IDE do a search for `_PROP` and you'll find a few hundred examples. In short, we frequently use a string to avoid mangling, but assign that string to a constant, and use the constant for indexing.

Reply:
I don't see why we don't want those key/value pairs typed

Reply:
Thanks for fixing this!